src/avram/tasks/db/rollback_to.cr

```diff
@@ -0,0 +1,16 @@
+require "colorize"
+
+class Db::RollbackTo < LuckyCli::Task
+  summary "Rollback to a specific migration"
+
+  def call(version : String? = nil)
```

Comment:
It looks like this argument is not being used so I think it can be removed

Reply:
I copied the pattern from `gen.migration`. I was assuming there was some reason to provide both a method argument and from ARGV in general. I'll remove it.

Reply:
Probably left on accident :S I think we can use something like this to test with in the future though. I need to go through and cleanup specs and do some refactoring at some point :)

Reply:
So should I leave it in in case of future test or leave it out for now? I'm happy both ways :)

Reply:
Sorry I wasn't clear there at all 🤣  you can go ahead and  remove it for now. I'll add it back in when I spruce up the tests